service/jobmanager/jobmanagerv1/jobmanager.proto

```diff
@@ -115,12 +115,15 @@ message JobStatusList {
 // The OutputStream enumeration captures the set of output stream
 // the JobManager can stream from the process.
 enum OutputStream {
+    // The unset value
+    UNSET = 0;
```

Comment:
FYI: Proto enums are package/namespace-scoped when compiled to certain languages, so typical style is to avoid overly generic names like "UNSET" and go for more verbose options like `MY_ENUM_NAME_UNSPECIFIED = 0`.